features/org.wso2.carbon.consent.mgt.server.feature/resources/dbscripts/mysql-cluster.sql

```diff
@@ -69,31 +69,39 @@ CREATE TABLE CM_SP_PURPOSE_ASSOC (
 ) ENGINE NDB;
 
 CREATE TABLE CM_SP_PURPOSE_PURPOSE_CAT_ASSC (
+  CM_SP_PURPOSE_PURPOSE_CAT_ASSC_ID INTEGER NOT NULL AUTO_INCREMENT,
```

Comment:
Any specific reason to introduce primary key column only for mysql-cluster scripts? Keeping the DDL queries consistent across different DB types would make them more maintainable.